vissl/hooks/nnclr_hooks.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import logging
+
+import torch
+from classy_vision import tasks
+from classy_vision.hooks.classy_hook import ClassyHook
+from vissl.models.heads import MLP
+from vissl.utils.env import get_machine_local_and_dist_rank
+
+
+class NNCLRHook(ClassyHook):
```

Comment:
@prigoyal,

I'm not sure if this is the best way of creating a predictor head.